gnocchi/tests/indexer/sqlalchemy/test_migrations.py

```diff
@@ -49,6 +49,20 @@ def setUp(self):
         self.index = indexer.get_driver(self.conf)
         self.index.upgrade(nocreate=True)
         self.addCleanup(self._drop_database)
+        # NOTE(sileht): remove tables dynamically created by other tests
```

Comment:
That may need to be rebased, mine have been merged.

Reply:
ack